Makefile

```diff
@@ -1,11 +1,53 @@
-.PHONY: fmt
-fmt: ## Run go fmt against code.
-	go fmt ./...
+GO_VERSION ?= 1.20.2
+GOLANG_CI_VER ?= v1.52
+GOSEC_VER ?= 2.15.0
+TEST_COVERAGE_FILE=lcov.info
+TEST_COVERAGE_HTML_FILE=coverage_unit.html
+TEST_COVERAGE_FUNC_FILE=func_coverage.out
 
-.PHONY: vet
-vet: ## Run go vet against code.
-	go vet ./...
+# CONTAINER_RUNNABLE checks if tests and lint check can be run inside container.
+PODMAN ?= $(shell podman -v > /dev/null 2>&1; echo $$?)
+ifeq ($(PODMAN), 0)
+CONTAINER_RUNTIME=podman
+else
+CONTAINER_RUNTIME=docker
+endif
+CONTAINER_RUNNABLE ?= $(shell $(CONTAINER_RUNTIME) -v > /dev/null 2>&1; echo $$?)
 
-.PHONY: test
-test: fmt vet ## Run tests.
-	go test ./... -coverprofile cover.out
\ No newline at end of file
+.PHONY: unit_clean
```

Comment:
In order to change Makefile commands you have to change PROW job as well. So for example you want to add 'make unit_clean' you have to add it in .prow.yaml for example like this:
```
  - name: presubmit-nephio-go-unit-clean
    decorate: true
    run_if_changed: "(\\.go)$"
    spec:
      containers:
      - image: golang:1.20.2
        command:
        - make
        args:
        - unit_clean
```

Reply:
@radoslawc Thanks, I shall update the PR